package.json

```diff
@@ -10,11 +10,17 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "axios": "^1.7.2",
```

Comment:
do you use axios anywhere in your application? If not, you can uninstall it by running 
`npm uninstall axios` in your terminal